### ISSUE TITLE
Documentation: Rewrite the Airbyte OSS Quickstart

### ISSUE THREAD

**Issue by Amruta-Ranade** [airbytehq/airbyte#15482]
**Note:** If you'd like to work on this issue, please read the [Docs Contributor guide](https://docs.airbyte.com/contributing-to-airbyte/updating-documentation) and comment on the issue to confirm it is still up for grabs.

The current [Airbyte OSS Quickstart](https://docs.airbyte.com/category/airbyte-open-source-quickstart) isn't user-centric since it spans several pages that requires multiple clicks to get all the information. It also overuses screenshots instead of a well-documented procedure. It also doesn't follow the [style guide](https://developers.google.com/style) we use for other docs.

Tasks:
- Convert the multi-page quickstart into a one-pager (similar to the [Airbyte Cloud quickstart](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud))
- Ensure it adheres to the Google style guide

**Comment by mirasdar** [comment]
Hi @Amruta-Ranade  I will like to work on this. Just last week I got a cash prize for completing a quick install guide for my company kyndryl in a bugathon. Our main guide is 240 pages and I got it to 14 pages for a person with basic DR understanding.

**Comment by Amruta-Ranade** [comment]
@mirasdar Sure! Please comment on this issue with a list of changes you'd make and we can take it from there :)
Here's our updated Contributing Guide for reference: https://docs.airbyte.com/contributing-to-airbyte/updating-documentation

**Comment by mirasdar** [comment]
@Amruta-Ranade I propose to follow the structure 
 Title Airbyte OSS Quickstart 
 Set up your Airbyte OSS
 Set up a source 
 Set up a destination 
 Set up a connection 
 Verify the connection 
 Allowlist IP Address (will check if appplicable else delete this section)  
1.Will ensure google style guide is followed 
2. Will replace screenshots with 3 or 4 steps 
3. Will createa decluttered  to do list of steps and provide links to all other cases at bottom 
4. IMHO verify connection link needs to be added in Quickstart to verify. 
5. Will try to fit all this in 1 page (my draft version is 1.5 word pages and what to delete to shorten with out loss of info is achallenge for me  )

**Comment by Amruta-Ranade** [comment]
Sounds good! Assigning the issue to you. 

Feel free to open a PR with the updated draft. I'm also happy to review your draft before you open a PR if you prefer. In that case, please share your draft in a Google doc with the Suggesting mode turned on :)

**Comment by mirasdar** [comment]
@Amruta-Ranade  please find the link below 
https://docs.google.com/document/d/1mDFtwkbWoxj5L9JOQV7pvMAxet8zdZGsS7h99JMVigI/edit?usp=sharing

**Comment by mirasdar** [comment]
@Amruta-Ranade  Updated the doc as per review comments please find the link below 
https://docs.google.com/document/d/1KtZNsiFtwHPq2mJFwDOnb1iYEDl3OsdH/edit#heading=h.gjdgxs

**Comment by Amruta-Ranade** [comment]
@mirasdar Just checking in to see if we have an update on the PR?

**Comment by Amruta-Ranade** [comment]
@mirasdar Last call for the PR! Will have to reassign the issue by next week

**Comment by mirasdar** [comment]
Hi @***@***.*** ***@***.***>  I am working on creation of PR.
I am doing this first time and faced long file name issue in windows so
moved to installing Linux wsl2 and Docker desktop. I am still facing
challenges in forking a branch and complete the process. I am trying g to
resolve this issue to complete the PR creation.   The quick install doc
changes as per review comments are completed.
Regards,
Niranjan

**Comment by Amruta-Ranade** [comment]
Hi, you can edit the file directly from GitHub: https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files

**Comment by mirasdar** [comment]
Hi Amruta,
sorry for the gap in communication from my side. I was hospitalized and had
a surgery and was in recovery most of the month.
I did minimal communication to IBM/kyndryl manager about my health
emergency.
I opened my gmail after weeks and missed out your earlier email.
I missed out on communicating with you for the WIP issue opened and left
hanging. I have recovered now and back to work partially with breaks.
I should have communicated with you. You have been very supportive and
patient with me and I thank you for that.
I will be back to almost normal health in about 2 weeks. I look forward to
work with Airbyte and completing the assigned task next time.

Regards,
Niranjan
